src/Tests/Microsoft.NET.Perf.Tests/PerfTest.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Microsoft.Xunit.Performance.Api;
+using Xunit;
+
+//  Don't run any perf tests in parallel
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly)]
+
+namespace Microsoft.NET.Perf.Tests
+{
+    public class PerfTest
+    {
+        public static int DefaultIterations { get; set; } = 1;
+
+        public string ScenarioName { get; set; }
+        public string TestName { get; set; }
+        public int NumberOfIterations { get; set; } = DefaultIterations;
+        public TimeSpan Timeout { get; set; } = TimeSpan.FromMinutes(20);
+        public ProcessStartInfo ProcessToMeasure { get; set; }
+        public string TestFolder { get; set; }
+
+        public bool GetPerformanceSummary { get; set; } = true;
+        public bool GetBinLog { get; set; }
+
+        public void Run([CallerMemberName] string callerName = null)
+        {
+            //  Handle case where we're running inside VS (or via dotnet test), so the Main method hasn't run to initialize the perf harness
+            //  In the future we may want to do this via an xUnit fixture, which would also let us call the dispose method to write the results
+            //  afterwards
+            if (_performanceHarness == null)
+            {
+                Program.HandlePerfArgs(new List<string>()
+                {
+                    "--iterations",
+                    "1"
+                });
+            }
+
+            TestName = TestName ?? callerName;
+
+            Stopwatch stopwatch = new Stopwatch();
+            TimeSpan[] executionTimes = new TimeSpan[NumberOfIterations];
```

Comment:
The `stopwatch` and `executionTimes` variables are left over from previous versions of the code and should now be deleted.

Reply:
Fixed...